docs/setup-cluster/k8s/resource-caps.rst

```diff
@@ -23,7 +23,8 @@ bindings and resource quotas using either the WebUI or the CLI.
    specify a namespace in the namespace field, the workspace is bound to that namespace. If the
    field is left blank, the workspace is bound to the namespace specified in the
    ``resource_manager.default_namespace`` section of your master configuration (when set), and is
-   otherwise bound to the default Kubernetes "default" namespace.
+   otherwise bound to the release namespace. We fall back to the default Kubernetes namespace,
```

Comment:
```suggestion
otherwise bound to the release namespace. For non-helm Determined deployments, it falls back to the default Kubernetes namespace, ``default``.
```

Reply:
fixed!